admin/apple-actions/index/class-push.php

```diff
@@ -233,6 +233,57 @@ private function push( $user_id = null ) {
 			);
 		}
 
+		// Special logic only if autosync push is enabled.
+		if ( $this->settings->api_autosync ) {
+			// Get the list of term IDs that should trigger a skip push from plugin settings.
+			$skip_term_ids = json_decode( $this->settings->api_autosync_skip );
+			if ( ! is_array( $skip_term_ids ) ) {
+				$skip_term_ids = [];
+			}
+
+			/**
+			 * Filters whether the post should be skipped and not pushed to Apple News
+			 * based on taxonomy term IDs that are associated with the post.
+			 *
+			 * Allows you to stop publication of a post to Apple News based on whether a
+			 * certain taxonomy term ID is applied to the post. A common use case is to
+			 * not publish posts with a certain category or tag. The default value for
+			 * this filter is the value of the skip push term IDs from the API settings
+			 * for the plugin, but the list can be modified for individual posts via
+			 * this filter.
+			 *
+			 * @since 2.3.0
+			 *
+			 * @param int[] $term_ids The list of term IDs that should trigger a skipped push. Defaults to the term IDs set in plugin options.
+			 * @param int   $post_id  The ID of the post being exported.
+			 */
+			$skip_term_ids = apply_filters( 'apple_news_skip_push_term_ids', $skip_term_ids, $this->id );
```

Comment:
Added this to the wiki, I'd make sure it's accurate. https://github.com/alleyinteractive/apple-news/wiki/Filter:-apple_news_skip_push_term_ids

Reply:
It was pretty close - I just had to update the data type of the first parameter (which is an integer array in this case, in contrast with a boolean for the skip_push filter). I also saw that you added it to the action and filter hook index, which is perfect. Thanks!